README.md

```diff
@@ -124,20 +125,19 @@ template. For more information, see the [example](./examples/splunk-wrapper/READ
    For more information about setting environment variables in the AWS console,
    see [Using AWS Lambda environment variables](https://docs.aws.amazon.com/lambda/latest/dg/configuration-envvars.html)
    on the AWS website.
-5. By default, the Splunk Lambda wrapper uses B3 context propagation. 
+5. By default, the Splunk Lambda wrapper uses `tracecontext,baggage` context propagation. 
```

Comment:
same comment here about this not being meaningful to end-users.

Reply:
+1